src/server/scripts/Commands/cs_spy.cpp

```diff
@@ -0,0 +1,279 @@
+                        return true;
+                    }
+
+                    gmListeningGroup[group->GetLowGUID()] = handler->GetSession()->GetPlayer()->GetGUIDLow();
+                    handler->PSendSysMessage(LANG_COMMAND_SPY_FOLLOWING_GROUP, target->GetName().c_str());
+                }
+            }
+            else
+                handler->PSendSysMessage(LANG_NOT_IN_GROUP, target->GetName().c_str());
+        }
+        else
+            handler->SendSysMessage(LANG_PLAYER_NOT_FOUND);
+
+        return true;
+    }
+
+    static bool HandleSpyUnFollowCommand(ChatHandler* handler, const char* args)
+    {
+        if(!*args)
+            return false;
+
+        char *cName = strtok ((char*)args," ");
+        if(!cName)
+            return false;
+
+        std::string pTarget = args;
+        if(!normalizePlayerName(pTarget))
+        {
+            handler->SendSysMessage(LANG_PLAYER_NOT_FOUND);
+            return true;
+        }
+
+        if(Player* target = ObjectAccessor::FindPlayer(sObjectMgr->GetPlayerGUIDByName(pTarget.c_str())))
+        {
+            handler->PSendSysMessage(LANG_COMMAND_SPY_UNFOLLOW, cName);
+            if(gmListening.find(target->GetGUIDLow()) != gmListening.end())
+                gmListening.erase(target->GetGUIDLow());
+        }
+        else
+            handler->SendSysMessage(LANG_PLAYER_NOT_FOUND);
+        return true;
+    }
+
+    static bool HandleSpyUnFollowGroupCommand(ChatHandler* handler, const char* args)
+    {
+        if(!*args)
+            return false;
+
+        char *cName = strtok ((char*)args," ");
+        if(!cName)
+            return false;
+
+        std::string pTarget = args;
+        if(!normalizePlayerName(pTarget))
+        {
+            handler->SendSysMessage(LANG_PLAYER_NOT_FOUND);
+            return true;
+        }
+
+        if(Player* target = ObjectAccessor::FindPlayer(sObjectMgr->GetPlayerGUIDByName(pTarget.c_str())))
+        {
+            if(Group* group = target->GetGroup())
+            {
+                if(gmListeningGroup.find(group->GetLowGUID()) != gmListening.end())
+                {
+                    gmListeningGroup.erase(group->GetLowGUID());
+                    handler->PSendSysMessage(LANG_COMMAND_SPY_UNFOLLOW_GROUP, cName);
+                }
+                else
+                    handler->PSendSysMessage(LANG_COMMAND_SPY_NOT_FOLLOWING_GROUP, cName);
+            }
+            else
+                handler->PSendSysMessage(LANG_NOT_IN_GROUP, target->GetName().c_str());
+        }
+        else
+            handler->SendSysMessage(LANG_PLAYER_NOT_FOUND);
+        return true;
+    }
+
+    static bool HandleSpyClearCommand(ChatHandler* handler, const char* /*args*/)
+    {
+        gmListening.clear();
+        handler->SendSysMessage(LANG_COMMAND_SPY_CLEARED);
+        return true;
+    }
+
+    static bool HandleSpyStatusCommand(ChatHandler* handler, const char* args)
```

Comment:
```cpp
static bool HandleSpyStatusCommand(ChatHandler* handler, const char* /*args*/)
```

Reply:
Seems like I'm using less strict compiler flags :sweat_smile:

Reply:
Deku, do you edit it? (i don't edit anything you do else you get angry xD)

Reply:
I did. Sorry, but I was busy 😬